<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLocatedPage</class>
 <widget class="QWidget" name="CLocatedPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidgetPage1">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Coord system:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QComboBox" name="coordSystemComboBox">
         <item>
          <property name="text">
           <string>Particles are local to the system</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Particles are in world</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>User</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="disgradeWithLODCheckBox">
         <property name="text">
          <string>Disgrade with lod</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="parametricMotionCheckBox">
         <property name="text">
          <string>Parametric motion</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Max number of particles:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="NLQT::CEditRangeUIntWidget" name="maxNumParticleWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="setCurrentCountPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Set current count particles</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <spacer name="horizontalSpacer_20">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>61</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>224</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWidgetPage2">
      <attribute name="title">
       <string>Life time</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="limitedLifeTimeCheckBox">
         <property name="text">
          <string>Limited life time</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="trigerOnDeathCheckBox">
            <property name="text">
             <string>Triger on death</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="editPushButton">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>251</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="NLQT::CAttribFloatWidget" name="lifeWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWidgetPage3">
      <attribute name="title">
       <string>Particle mass</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>274</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="NLQT::CAttribFloatWidget" name="massWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLQT::CEditRangeUIntWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CAttribFloatWidget</class>
   <extends>QGroupBox</extends>
   <header>attrib_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
